### ISSUE TITLE
[Bug]: Servant won't return and doesn't register lord

### ISSUE THREAD

**Issue by LoafDead** [TeamLapen/Vampirism#1048]
### Minecraft Version

1.17.x

### Forge Version

37.1.1

### Vampirism Version

1.8.4-beta

### Singleplayer/Multiplayer

Server

### What happened?

I'm on a 1.17.1 server running a custom mod pack. I had a vampire servant but whenever i try to call them back to me It doesn't work. the console says "Retrieved entity is not a minion entity null" and "Minion without lord.". whenever I try to summon them they show up with the stock vampire skin and their name for an instant then disappear. I don't know if this is relevant but I increased my lord level so maybe that did something. my ranking is duke but I think I got the minion when I was a lower rank.

### Relevant log output

```shell
[00:01:03] [Server thread/WARN]: Retrieved entity is not a minion entity null
[00:01:03] [Server thread/WARN]: Retrieved entity is not a minion entity null
[00:01:03] [Server thread/WARN]: Retrieved entity is not a minion entity null
[00:01:03] [Server thread/WARN]: Minion without lord.
```

### Other relevant mods

[Caelus API (Forge)]()
TheIllusiveC4
caelus-forge-1.17.1-3.0.0.1.jar

[Cloth Config API (Forge)]()
shedaniel
cloth-config-5.2.47-forge.jar

[XP Tome]()
bl4ckscor3
xptome-1.17.1-v2.1.3.jar

[Advanced Netherite]()
AutovwLive
advancednetherite-1.10.0-1.17.1.jar

[Advanced Shulkerboxes]()
henkelmax
shulkerbox-1.17.1-1.0.0.jar

[AI Improvements]()
QueenOfMissiles
AI-Improvements-1.17.1-0.4.0.jar

[Alex's Mobs]()
sbom_xela
alexsmobs-1.14.1.jar

[Ambient Environment]()
Jaredlll08
AmbientEnvironment-forge-1.17.1-6.0.22.jar

[AmbientSounds 5]()
CreativeMD
AmbientSounds_v4.0.4_mc1.17.1.jar

[Aquatic Torches]()
realmayus
aquatictorches-1.17-1.0.0.jar

[Armor Points ++ / Health Stacking]()
CheosTwitch
armorpointspp-forge-1.17.1-2.1.3.jar

[Artifacts]()
ochotonida
artifacts-1.17.1-3.0.2.jar

[Bag of Yurting]()
Commoble
bagofyurting-1.17.1-2.0.0.0.jar

[Balm (Forge Edition)]()
BlayTheNinth
balm-1.8.2+0.jar

[Better Ore Generation [FORGE]]()
KingpvzYT
17-betteroregeneration_1.5.jar

[Builders Crafts & Additions (Forge)]()
MRHminer
buildersaddition-1.17.1-20211105a.jar

[Castle in the Sky]()
songxia23
castle_in_the_sky-1.17.1-0.1.1.jar

[Citadel]()
sbom_xela
citadel-1.9.5-1.17.1.jar

[Clumps]()
Jaredlll08
Clumps-forge-1.17.1-7.0.10.jar

[Comforts (Forge)]()
TheIllusiveC4
comforts-forge-1.17.1-5.0.0.1.jar

[Corpse]()
henkelmax
corpse-1.17.1-1.0.7.jar

[Cosmetic Armor Reworked]()
LainMI
CosmeticArmorReworked-1.17.1-v1b.jar

[CreativeCore]()
CreativeMD
CreativeCore_v2.3.0_mc1.17.1.jar

[Curios API (Forge)]()
TheIllusiveC4
curios-forge-1.17.1-5.0.2.7.jar

[Curious Elytra (Forge)]()
TheIllusiveC4
curiouselytra-forge-1.17.1-5.0.0.1.jar

[Dark Paintings]()
DarkhaxDev
DarkPaintings-Forge-1.17.1-7.0.3.jar

[Decorative Blocks]()
stohun
Decorative Blocks-forge-1.17.1-1.8.0.jar

[Desolation (Forge)]()
y4z0n
Desolation 1.1.0-2 1.17.1.jar

[Dimensional Dungeons]()
Catastrophe573
dimdungeons-1.13.1-forge-1.17.1.jar

[Doggy Talents]()
percivalalb
DoggyTalents-1.17.1-2.2.10.jar

[Dungeon Crawl]()
xiroc_
DungeonCrawl-1.17.1-2.3.5.jar

[Dynamic Lights]()
atomicstrykergrumpy
dynamiclights-1.17.1.2.jar

[Enhanced Celestials - Blood, Blue, & Harvest (Super) Moons]()
Corgi_Taco
enhancedcelestials-2.0.9-1.17.1.jar

[ExpandAbility]()
florensie
expandability-4.0.2-forge.jar

[Explorer's Compass]()
Chaosyr
ExplorersCompass-1.17.1-1.1.2-forge.jar

[Fast Leaf Decay]()
olafskiii
FastLeafDecay-26.2.jar

[Frosted Friends]()
Furti_Two
frosted-friends-1.17.1-1.0.6.jar

[Goblin Traders]()
MrCrayfish
goblintraders-1.6.3-1.17.1.jar

[Godly Vampirism]()
maxanier
GodlyVampirism-1.17.1-1.0.3-beta.1.jar

[Guide-API Village and Pillage]()
maxanier
Guide-API-VP-1.17.1-2.2.3.jar

[Hud Compass]()
gigaherz
HudCompass-1.17.1-0.2.2.jar

[Inventory HUD+]()
dmitrylovin
InventoryHud_[1.17.1].forge-3.4.1.jar

[Jousting]()
stepcros
jousting-1.17.1-v1.0.4.jar

[Just Enough Items (JEI)]()
mezz
jei-1.17.1-8.3.1.62.jar

[Just Enough Resources (JER)]()
way2muchnoise
JustEnoughResources-1.17.1-0.13.1.134.jar

[Kobolds!]()
Jusey1z
Kobolds-1.6.1.jar

[Let Sleeping Dogs Lie]()
ohaiiChun
LetSleepingDogsLie-1.17.1-1.1.1.jar

[Long Fall Boots]()
Nano_Heart
longfallboots-1.7.0a.jar

[Lucent]()
ModdingLegacy
lucent-1.17.1-1.1.1.jar

[Macaw's Bridges]()
sketch_macaw
mcw-bridges-2.0.1-mc1.17.1.jar

[Macaw's Doors]()
sketch_macaw
mcw-doors-1.0.5-mc1.17.1.jar

[Macaw's Fences and Walls]()
sketch_macaw
mcw-fences-1.0.2-mc1.17.1.jar

[Macaw's Lights and Lamps]()
sketch_macaw
mcw-lights-1.0.2-mc1.17.1.jar

[Macaw's Paintings]()
sketch_macaw
mcw-paintings-1.0.2-mc1.17.1.jar

[Macaw's Roofs]()
sketch_macaw
mcw-roofs-2.1.0-mc1.17.1.jar

[Macaw's Trapdoors]()
sketch_macaw
mcw-trapdoors-1.0.4-mc1.17.1.jar

[Macaw's Windows]()
sketch_macaw
mcw-windows-2.0.2-mc1.17.1.jar

[Mahou Tsukai]()
stepcros
mahoutsukai-1.17.1-v1.33.3.jar

[MmmMmmMmmMmm (Target Dummy)]()
MehVahdJukaar
MmmMmmMmmMmm-1.17-1.5.0.jar

[More Ores In ONE]()
KG20
MoreOresInONE-1.0_1.17.1.jar

[MrCrayfish's Furniture Mod]()
MrCrayfish
cfm-7.0.0pre25-1.17.1.jar

[Nature's Compass]()
Chaosyr
NaturesCompass-1.17.1-1.9.2-forge.jar

[Nether's Exoticism]()
Furti_Two
nether-s-exoticism-1.17.1-1.1.3.jar

[Ore Excavation]()
Funwayguy
OreExcavation-1.9.158.jar

[Placebo]()
Shadows_of_Fire
Placebo-1.17.1-5.0.2.jar

[Rare Ice (Forge/Fabric)]()
shedaniel
rare-ice-0.3.0.jar

[Reroll]()
Draylar1
reroll-1.3.0-1.17.1.jar
This version is incompatible
with the profile modloader
(Forge). [Learn more](https://support.curseforge.com/en/support/solutions/articles/9000202116-modloaders-forge-and-fabric-)

[Serene Seasons]()
TheAdubbz
SereneSeasons-1.17.1-5.0.0.6-universal.jar

[Slimy Stuff]()
Furti_Two
slimy-stuff-1.17.1-1.0.6.jar

[Sons Of Sins]()
Furti_Two
sons-of-sins-1.17.1-1.0.9.jar

[Sophisticated Backpacks]()
P3pp3rF1y
sophisticatedbackpacks-1.17.1-3.12.3.496.jar

[Stalwart Dungeons]()
Furti_Two
stalwart-dungeons-1.17.1-1.2.0.jar

[Stoneholm, Underground Villages (Forge)]()
TheGrimsey
stoneholm-1.3.jar

[SuperTools]()
KG20
SuperTools-1.0_1.17.1.jar

[SwingThroughGrass]()
exidex
swingthroughgrass-1.17.1-1.6.0.jar

[The Twilight Forest]()
Benimatic
twilightforest-1.17.1-4.0.875-universal.jar

[True Darkness]()
GrondagTheBarbarian
darkness-forge-mc117-2.0.87.jar

[Tumbleweed]()
bconv
tumbleweed-1.17-0.4.10.jar

[Useless Sword]()
Furti_Two
useless-sword-1.17.1-V1.3.9.jar

[Valhelsia Core]()
ValhelsiaTeam
valhelsia_core-1.17.1-0.3.0.jar

[Valhelsia Structures]()
ValhelsiaTeam
valhelsia_structures-1.17.1-0.1.1.jar

[Vampires Need Umbrellas]()
Focamacho
VampiresNeedUmbrellas-1.17.1-1.1.jar

[Vampirism - Become a vampire!]()
maxanier
Vampirism-1.17.1-1.8.4-beta.1.jar

[Wandering Bag]()
Furti_Two
wandering-bag-1.17.1-1.0.3.jar

[Waystones]()
BlayTheNinth
waystones-8.2.0+0.jar

[Wither Skeleton Tweaks]()
Shadows_of_Fire
WitherSkeletonTweaks-1.17.1-6.0.0.jar

[WTHIT Forge Edition]()
badasintended
wthit-forge-3.11.2.jar

[Xaero's Minimap]()
xaero96
Xaeros_Minimap_22.1.2_Forge_1.17.1.jar

[Xaero's World Map]()
xaero96
XaerosWorldMap_1.20.1_Forge_1.17.1.jar

**Comment by LoafDead** [comment]
restarted my server and the problem solved itself still might wanna fix the bug tho

**Comment by maxanier** [comment]
Thank you for reporting.
This is likely a difficult bug to narrow down.

Please let me know if you (or anyone else) notice this frequently, otherwise I will probably not put too much time into this.

**Comment by Cheaterpaul** [comment]
Duplicate of #1093

**Comment by maxanier** [comment]
Fixed for 1.18. 
Unfortunately, 1.17 is not really supported by us anymore
